tests/helper/grafana_helper.js

```diff
@@ -7,6 +7,20 @@ class Grafana extends Helper {
   constructor(config) {
     super(config);
     this.resultFilesFolder = `${global.output_dir}/`;
+    this.signInWithSSOButton = '//a[contains(@href,"login/generic_oauth")]';
```

Comment:
as I see we don't use anything special from playwright native API right now, at least until we want to set the session via cookies and local storage, do think it would be better to move those locators to a page class?